extensions/wikia/AdEngine/js/video/player/porvata/googleIma.js

```diff
@@ -2,7 +2,7 @@
 define('ext.wikia.adEngine.video.player.porvata.googleIma', [
 	'ext.wikia.adEngine.utils.scriptLoader',
 	'ext.wikia.adEngine.video.player.porvata.googleImaPlayerFactory',
-	'ext.wikia.aRecoveryEngine.adBlockRecovery',
+	'ext.wikia.aRecoveryEngine.sourcePoint.recovery',
 	'wikia.log',
 	'wikia.window'
 ], function (scriptLoader, imaPlayerFactory, adBlockRecovery, log, win) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Function has 5 parameters which is greater than 4 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)